app/models/message.rb

```diff
@@ -26,7 +27,7 @@ def text # rubocop:disable Metrics/AbcSize
       when 'group_declined'
         I18n.t('groups.send_deny_access_message.message', user: sender.name, group: Group.find(param_id).name)
       when 'collection'
-        I18n.t('collections.share_message.text', user: sender.name, collection: Collection.find(param_id).title)
+        I18n.t('collections.share_message.text', user: sender.name, collection: Collection.find_by(id: param_id)&.title)
```

Comment:
I've debugged the broken model test and made another proposal. As soon as we have a proper polymorphic association, the test will pass without any manual modifications.

Do you agree to the temporary solution I pushed? If so, I will just proceed merging, so that we can continue in #1679.

Reply:
Yes, that looks good :+1: